tests/stress/stress_test.go

```diff
@@ -641,3 +642,44 @@ func TestSync_SubmitExtrinsicLoad(t *testing.T) {
 		// Output: Balance transferred from Alice to Bob: 100000000000000
 	}
 }
+
+func TestStress_SecondarySlotProduction(t *testing.T) {
+	numNodes := 3
+
+	nodes, err := utils.InitializeAndStartNodes(t, numNodes, utils.GenesisThreeAuths, utils.ConfigDefault)
+	require.NoError(t, err)
+
+	primaryCount := 0
+	secondaryCount := 0
+	otherCount := 0
+	for i := 1; i < 20; i++ {
+		fmt.Printf("%d iteration\n", i)
```

Comment:
```suggestion
```

Reply:
this is a long test. Knowing the iteration number if useful.